assemblyline/odm/models/ontology/ontology.py

```diff
@@ -5,7 +5,7 @@
 from assemblyline.odm.models.ontology.filetypes import PE
 
 Classification = forge.get_classification()
-ODM_VERSION = "1.6"
+ODM_VERSION = "1.7"
```

Comment:
this is an opportunity to introduce a change log regarding the ontology. this was requested by someone in the past

Reply:
That will likely be something we pipeline during builds, similar to the auto-generated docs. I have an idea of how we want to do that and show obvious things like additions and removals that would be ODM-wide.